host.go

```diff
@@ -695,4 +698,193 @@ func (h *Host) InitCommands(c *chat.Commands) {
 			return nil
 		},
 	})
+
+	c.Add(chat.Command{
+		// TODO: find a better name for reverify
```

Comment:
IMO reverify is a fine name. :)

Reply:
I have to admit, I dislike it less now than when I added that comment.